scene/gui/color_picker.cpp

```diff
@@ -1029,6 +1029,12 @@ void ColorPicker::_sample_draw() {
 
 		sample->draw_rect(rect_old, old_color);
 
+		if (!old_color.is_equal_approx(color)) {
+			// Draw a revert indicator to indicate that the old sample can be clicked to revert to this old color.
+			// Adapt icon color to the background color so that it's always visible.
+			sample->draw_texture(theme_cache.sample_revert, rect_old.size * 0.5 - theme_cache.sample_revert->get_size() * 0.5, old_color.get_v() < 0.5 ? Color(1, 1, 1) : (Color(0.1, 0.1, 0.1)));
```

Comment:
```suggestion
			sample->draw_texture(theme_cache.sample_revert, rect_old.size * 0.5 - theme_cache.sample_revert->get_size() * 0.5, Math::lerp(0.75f, old_color.get_luminance(), old_color.a) > 0.455 ? Color(1, 1, 1) : (Color(0.01, 0.01, 0.01)));
```
IMO use this for max contrast, otherwise this will be hard to see on dark reds.

Reply:
Thanks! Note that the symbol should be the other way around (i.e. the condition should be `< 0.455`).